### ISSUE TITLE
Hi, could I ask why this commit reverted

### ISSUE THREAD

**Issue by fankes** [ThePBone/RootlessJamesDSP#71]
https://github.com/ThePBone/RootlessJamesDSP/commit/2ff91908cdf8ca57e555b4c752ee1c0a17c68dce

**Comment by ThePBone** [comment]
Hi, the commit was not reverted. Your changes should still be in the code base.

I did some code refactoring lately, so I moved your code from the MainActivity.kt into BaseActivity.kt (https://github.com/ThePBone/RootlessJamesDSP/blob/062cce2d9ccabf2aa30a23db4ad2cad5c1fe7dff/app/src/main/java/me/timschneeberger/rootlessjamesdsp/activity/BaseActivity.kt#L48). By putting it into BaseActivity, it will ensure that the 'Remove from recents' code is triggered even when other activities than MainActivity are launched.

**Comment by fankes** [comment]
But the switch function in the app behaviour is gone

**Comment by ThePBone** [comment]
Oops. I found the problem. It looks like the switch function is only visible in rootless builds and invisible in root builds because of some changes and rearrangements I made for the settings screen.

Commit 20c1d2ca4830610595b76239b1bf8be564711930 (included in version 1.4.3) fixes this issue. Root build APK: https://nightly.timschneeberger.me/rootlessjamesdsp-rootfull/JamesDSP-v1.4.3-30-root-full-universal-release-signed.apk
